pkg/daemon/server/service/rater/rater.go

```diff
@@ -230,11 +270,51 @@ func (r *Rater) getPodReadCounts(vertexName, podName string) *PodReadCount {
 	textParser := expfmt.TextParser{}
 	result, err := textParser.TextToMetricFamilies(resp.Body)
 	if err != nil {
-		r.log.Errorf("[vertex name %s, pod name %s]: failed parsing to prometheus metric families, %v", vertexName, podName, err.Error())
+		r.log.Errorf("[Pod name %s]:  failed parsing to prometheus metric families, %v", podName, err.Error())
```

Comment:
minor: `r.log.Errorf("[Pod name %s]: failed parsing to prometheus metric families, %v", podName, err)`